README.md

```diff
@@ -31,6 +31,15 @@ It allows users to:
 
    Apps directory is set using the `WEB_ASSET_APPS_PATH` environment variable.
 
+### App Installation With [oCIS Deployment](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full)
+
+1. Copy [`deployments/mdpresentation-viewer.yml`](./deployments/mdpresentation-viewer.yml) into the [web_extensions](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/web_extension)
+subfolder of oCIS full deployment example.
+2. Add `MDVIEWER=:web_extensions/mdpresentation-viewer.yml` to the `## oCIS Web Extensions ##` section of the `.env` file of your installation (file is located in `deployments/examples/ocis_full`).\ Add that variable in the `COMPOSE_FILE` variable at the last line.
```

Comment:
```suggestion
2. Add `MDPRESENTATION_VIEWER=:web_extensions/mdpresentation-viewer.yml` to the `## oCIS Web Extensions ##` section of the `.env` file of your installation (file is located in `deployments/examples/ocis_full`).\ Add that variable in the `COMPOSE_FILE` variable at the last line.
```

Reply:
I think the suggestion [above](https://github.com/JankariTech/web-app-presentation-viewer/pull/91/files#r1872547976) is clearer